test/integration/admin/api/services_controller_test.rb

```diff
@@ -67,6 +67,14 @@ class TenantHostTest < ActionDispatch::IntegrationTest
 
     attr_reader :provider, :service
 
+    test 'index can be paginated' do
+      FactoryBot.create_list(:simple_service, 5, account: provider)
+      get admin_api_services_path(access_token: access_token_value, format: :json, per_page: 3, page: 2)
+      assert_response :success
+      response_service_ids = JSON.parse(response.body)['services'].map { |response_service| response_service.dig('service', 'id') }
+      assert_equal provider.accessible_services.order(:id).offset(3).limit(3).select(:id).map(&:id), response_service_ids
```

Comment:
```suggestion
      assert_equal provider.accessible_services.order(:id).offset(3).limit(3).pluck(:id), response_service_ids
```

Reply:
Unfortunately, this fails for Oracle 😓 
![image](https://user-images.githubusercontent.com/11318903/95868091-2bf36d00-0d6a-11eb-9901-fcbf3015c8d6.png)

I will revert this commit in another PR.